tutor/templates/build/openedx/Dockerfile

```diff
@@ -172,7 +172,8 @@ COPY --link --chown=$APP_USER_ID:$APP_USER_ID --from=nodejs-requirements /opened
 COPY --link --chown=$APP_USER_ID:$APP_USER_ID --from=nodejs-requirements /openedx/edx-platform/node_modules /openedx/node_modules
 
 # Symlink node_modules such that we can bind-mount the edx-platform repository
-RUN ln -s /openedx/node_modules /openedx/edx-platform/node_modules
+RUN rm -rf /openedx/edx-platform/node_modules && \
```

Comment:
Under what circumstances would /openedx/edx-platform/node_modules exist beforehand?

Reply:
Sorry about this.

I have been trying to build the `openedx-dev` image locally using the following command.

```
tutor images build openedx-dev
```

The command above actually passes the `--build-context` argument when it runs `docker buildx build...`

```
docker buildx build .... --build-context=edx-platform=/path/to/local/repo/edx-platform
```

In the end, I figured out that my local repo had a residual `node_modules` directory that was empty and causing the `openedx-dev` image build to fail.

After removing the `node_modules` directory in my local repo, the `openedx-dev` image builds successfully.

I'll go a head and close this PR.